OSP: add note about compressed image format (#16891)
openshift/openshift-docs: modules/installation-osp-creating-image.adoc

//Links not valid--release images aren't posted yet.
. Download the latest {op-system} image from the https://access.redhat.com/downloads/content/290[Product Downloads] page of the Red Hat customer portal or the https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/4.2/[{op-system} image mirror page].

> 

Reviewer: remove this line

+
[NOTE]

> The OpenStack QCOW2 images are delivered in compressed format.  If you are using `curl` or `wget` to download the images, you'll need to provide the proper options when retrieving the images.  For `curl`, use `curl --compressed -J -L -O <url to OpenStack QCOW2 image>`.  For `wget`, use `wget --compression=auto <url to OpenStackQ QCOW2 image>`.

Reviewer: ```suggestion
The OpenStack QCOW2 images are delivered in compressed format, so you must specify additional options to download them. With `curl`, specify `curl --compressed -J -L -O <image_url>`.  With `wget`, specify `wget --compression=auto <image_url>`.
```

+

> 

Reviewer: remove this line
